src/header/header.scss

```diff
@@ -83,7 +83,7 @@
     width: 1px;
     height: $iui-baseline * 2;
     @include themed {
-      background-color: t(iui-color-background-4);
+      background-color: t(iui-color-background-border);
```

Comment:
background color as border color.. does not make sense

Reply:
In this case it does kind of make sense.  This element is `.iui-divider` which is basically replicating a border.  @mayank99 and I both found an article that explained why it's better to use a 1px wide element VS a `border` when inside a flexbox.  I can't find that article atm.

Reply:
oh, yes.. you are right.
Other same cases are track and rail, which I assume also makes sense..